nuxeo-features/nuxeo-platform-oauth/src/test/java/org/nuxeo/ecm/platform/oauth/tests/OAuth2ChallengeFixture.java

```diff
@@ -124,10 +128,18 @@
 
     @Before
     public void initOAuthClient() {
-        // Client to make the requests like a "Client" as the OAuth2 RFC describes it
+        // Clients to make the requests like a "Client" as the OAuth2 RFC describes it
+        // Authenticated client for the /oauth2/authorize and /oauth2/authorize_submit endpoints
+        authenticatedClient = JerseyClientHelper.clientBuilder()
+                                                .setRedirectsEnabled(false)
+                                                .setCredentials("Administrator", "Administrator")
+                                                .build();
+        // Unauthenticated client for the /oauth2/token endpoint
         client = JerseyClientHelper.clientBuilder()
                                    .setRedirectsEnabled(false)
-                                   .setCredentials("Administrator", "Administrator")
+                                   .setConnectionRequestTimeout(500000)
+                                   .setSocketTimeout(500000)
+                                   .setConnectTimeout(500000)
```

Comment:
I don't think setting the timeouts are required in this test, especially since the default is already 60 seconds.

Reply:
Yes it was for debugging purpose.